### ISSUE TITLE
BUG: Projects, categories, profile pages are crashing

### ISSUE THREAD

**Issue by EugeneLitvinchuk** [graphprotocol/mission-control-curator#118]
Recently all requests are started being blocked by CORS policy, and as a result application crashes on every page(projects, categories, profile page)

![123](https://user-images.githubusercontent.com/32702989/96499384-ff51c080-1255-11eb-98e3-d1bcbd9a93be.gif)

**Comment by bobyD** [comment]
Great bug reporting design, but unfortunately, a duplicate of #77, #97, #107
